TicketBooking.Common/AppExceptions/RequestParam.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace TicketBooking.Service.Models
+{
+    public class RequestParam
```

Comment:
The name doesn't make sense for pagination.

Reply:
fixed